### ISSUE TITLE
[BUG] Always check diff between base branch and last commit

### ISSUE THREAD

**Issue by yoanmLf** [yoanm/github-action-deps-versions-checker#63]
**Describe the bug**
In case a merge master (for instance) happens and it doesn't bring any change on `composer.lock` because of another merge made before, app think there is no change.
However, there may be changes based on base branch (mostly if a prior merge of the branch which was then merged on master has been already done on the PR)
Example : 
```
BASE     ***************_*********⤵
Branch1 ******⤵*******⤴
Branch2 *******_*******************_
```

**To Reproduce**
See above

**Expected behavior**
A new check has to be triggered anyway
Basically, app must always check between base branch and last commit ("fast" check between last comment commit and last commit must be removed)

**Screenshots**

**Additional context**
